Certified Sail Extension (CSE) Examples/cse/mapField/v1/mapField_en_US.properties

```diff
@@ -0,0 +1,8 @@
+name=Map
+description=Map component which is backed by Google Maps. Allows a user drop a pin.
+parameter.location.name=Location
+parameter.location.description=Location to display a map for. May be a country, city, full address, or anything the Google Geocoding API supports (https://developers.google.com/maps/documentation/geocoding/intro).
+parameter.pin.name=Pin
+parameter.pin.description=One or more variables that are updated when a user adds a pin to the map. The value to be saved will be a dictionary representing the pin, including fields for latitude and longitude. Use a!save() to save a modified or alternative value to a variable.
+parameter.key.name=Google API Key
+parameter.key.description=Key used to activate the google maps JavaScript API
```

Comment:
Still probably want these reviewed to confirm that the docs are acceptable

Reply:
@jrank @amyhorton @ franklihuan FYI please review documentation wording here when you get the chance.